src/lib-rust/src/locator.rs

```diff
@@ -290,6 +297,25 @@ impl VelopackLocator {
     fn path_as_string(path: &PathBuf) -> String {
         path.to_string_lossy().to_string()
     }
+
+    fn get_or_create_staged_user_id(&self) -> String {
+        let packages_dir = self.get_packages_dir();
+        let beta_id_path = packages_dir.join(".betaId");
+        if beta_id_path.exists() {
+            info!("Found existing staged user id...");
+            if let Ok(beta_id) = std::fs::read_to_string(&beta_id_path) {
+                return beta_id;
+            }
+        }
+        let new_id = Uuid::new_v4();
+        std::fs::write(&beta_id_path, &new_id).expect("Unable to write .betaId file");
```

Comment:
looks like this line needs to get removed (as the fs::write is duplicated on the next line in a better way)

Reply:
Done